manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -142,7 +132,10 @@ controllerManager:
     runAsGroup: 999
     runAsNonRoot: true
     runAsUser: 1000
-  extraRules: []
```

Comment:
why was this removed?

Reply:
was not intentional, I will readd it asap